index.android.js

```diff
@@ -20,10 +20,4 @@ import { AppRegistry } from 'react-native';
 
 import Storybook from './storybook';
 
-// https://github.com/facebook/react-native/issues/15902
-import 'core-js/es6/symbol';
-import 'core-js/fn/map';
-import 'core-js/fn/set';
-import 'core-js/fn/array/find';
```

Comment:
I was getting an error that `core-js` couldn't be found, which is weird as it's definitely a dependency and I even installed it manually to be sure.

However, since RN 0.59 we don't need these anyway. Not sure why it suddenly broke now but removing them fixes it.